.zuul.yaml

```diff
@@ -212,6 +212,18 @@
             override-checkout: main
             required-projects:
               - github.com/openstack-k8s-operators/watcher-operator
+            irrelevant-files: &irrelevant_files_watcher
+            # From watcher irrelevant_files from watcher-grenade job job: https://github.com/openstack/watcher/blob/master/.zuul.yaml#L151
+            # - ^(test-|)requirements.txt$ It is needed to find dependency issue while building packages in content provider.
```

Comment:
i was going to question if we should trigger on requirement changes but i  see someone else ask that too :)